include/capstone/tricore.h

```diff
@@ -18,40 +18,43 @@ extern "C" {
 #pragma warning(disable : 4201)
 #endif
 
-//> Operand type for instruction's operands
+/// Operand type for instruction's operands
 typedef enum tricore_op_type {
-	TRICORE_OP_INVALID = 0, // = CS_OP_INVALID (Uninitialized).
-	TRICORE_OP_REG, // = CS_OP_REG (Register operand).
-	TRICORE_OP_IMM, // = CS_OP_IMM (Immediate operand).
-	TRICORE_OP_MEM, // = CS_OP_MEM (Memory operand).
+	TRICORE_OP_INVALID = 0, ///< CS_OP_INVALID (Uninitialized).
+	TRICORE_OP_REG, ///< CS_OP_REG (Register operand).
+	TRICORE_OP_IMM, ///< CS_OP_IMM (Immediate operand).
+	TRICORE_OP_MEM, ///< CS_OP_MEM (Memory operand).
```

Comment:
See comment above about memory operands. This must be `TRICORE = CS_OP_MEM = 0x80`.